cypress/support/api/apiTests.spec.js

```diff
@@ -0,0 +1,77 @@
+import {DELETE, GET, POST, PUT} from "cypress/support/api/apiUtils.js";
+import {
+    createDashboard,
+    deleteDashboardById,
+    getActivityItemId,
+    updateDashboardById
+} from "cypress/support/api/endpoints.js";
+
+describe('CRUD Tests', () => {
```

Comment:
- Please, add negative tests too (as your task requires).
- Add Logger.
- Configure proxy so that API calls can be caught by network sniffers.